### ISSUE TITLE
Alternate between reviewer backgrounds

### ISSUE THREAD

**Issue by galantra** [my-Anki/Beautify-Anki#10]
Hi,
my girlfriend has shown interest in Anki, and I want to customize it for her as much as possible. 
The reviewer backgrounds are an awesome opportunity. Would it be possible to assign several backgrounds to one deck, and alternate between them (preferably randomly)? For me it's actually not important if alternate within a group of pictures that is assigned to only one deck, or if the reviewer background just alternates randomly regardless of the deck. I guess that the latter would be simpler, and therefore easier to accomplish?

**Comment by ShoroukAziz** [comment]
It's possible to achieve both. Actually, having the background changes between a set of images for each deck doesn't even require writing an addon (as long as the deck has the same kind of notes). just some CSS, and JS maybe, in the note itself.
I'll help you achieve it just give me some time.

**Comment by galantra** [comment]
Thanks for making the priorities transparent!

Thanks also very much for giving the tip about CSS and JS. I hardly know anything about it, so I appreciate this tip.

I see you're receiving a lot of issues filed right now. If you don't mind, I think I'll ask a big community like reddit, as it is just a matter of CSS and JS, in order to take the pressure off of you. This will be sufficient for our purpose.

PS: Wait – maybe I misunderstood what the add-on affects. Does it affect also the deck background in the review screen (i.e., the purple-red-ish background in the gif on ankiweb), or only the background in the screen that is shown after you select a deck (i.e., the Eiffel tower picture in the gif on ankiweb)?

I now remember a blogpost of yours (shared on reddit, I believe) where you described how to beautify the background in the review screen (via the note template).

**Comment by ShoroukAziz** [comment]
> PS: Wait – maybe I misunderstood what the add-on affects. Does it affect also the deck background in the review screen (i.e., the purple-red-ish background in the gif on ankiweb), or only the background in the screen that is shown after you select a deck (i.e., the Eiffel tower picture in the gif on ankiweb)?

No, the addon doesn't affect the reviewer's background. That's just the template I am using this could be achieved with a single line of code really
In the Styling section of the card. find that part
```
.card {
 font-family: arial;
 font-size: 20px;
 text-align: center;
 color: black;
 background-color: white;
}
```
and add that line at the end
```
background: url("name of your picture");
```

so the code should look like this
```
.card {
 font-family: arial;
 font-size: 20px;
 text-align: center;
 color: black;
 background-color: white;
background: url("name of your picture");
}
```

**The picture should be in your collection.media folder. and rename it to start with '_' so it doesn't get deleted if you checked your media**

I'd really like to help with what you actually need. So, whenever I have the time I'll let you know. if you hadn't found out how to do it by then.

**Comment by coco1602** [comment]
> Hi,
> my girlfriend has shown interest in Anki, and I want to customize it for her as much as possible.
> The reviewer backgrounds are an awesome opportunity. Would it be possible to assign several backgrounds to one deck, and alternate between them (preferably randomly)? For me it's actually not important if alternate within a group of pictures that is assigned to only one deck, or if the reviewer background just alternates randomly regardless of the deck. Could you help me with that? I guess that the latter would be simpler, and therefore easier to accomplish?

Hi ! Some time ago, I tried to do the same thing (making a cycled background change during reviews) and I posted the code on my blog (https://anki-paces.blogspot.com/2018/01/avoir-un-fond-decran-dynamique-dans-anki.html).
I can help you to make it work, if you are still interested.

**Comment by galantra** [comment]
@coco1602 Thanks, I'll try it! Seems like it is easy to follow, with the checklist. I will give you feedback
